src/components/felles/minikomponenter.tsx

```diff
@@ -14,3 +14,19 @@ export const Errormelding = () => (
         </Alert>
     </div>
 );
+
+export const MidtstiltMelding = ({
```

Comment:
I midt hode er det overkill å lage en egen komponent bare for å få på en CSS-klasse, men den klassen som vi har fra før tilfredsstiller ikke behovet! Så mitt forslag er å heller bare lage en ny CSS-klasse som fikser problemet, altså midtstiller f.eks. en alert:
```css
.klassenavn {  
  margin: 0 auto;
  width: fit-content;
}
```